pdp/policy.go

```diff
@@ -63,12 +69,15 @@ func (p *Policy) describe() string {
 	ruleIdx := 0
 	for _, rule := range p.rules {
```

Comment:
You don't need to iterate over all rules. Find first nDisplayRuleLimit - 1 rules and then from the end find another one. Also you need to process case when number of rules less than nDisplayRuleLimit separately.